corehq/apps/app_manager/templates/app_manager/module_view_report.html

```diff
@@ -1,24 +1,73 @@
 {% extends "app_manager/managed_app.html" %}
 {% load xforms_extras %}
 {% load hq_shared_tags %}
+{% load reports_core_tags %}
 {% load i18n %}
+
+{% block head %}{{ block.super }}
+    <link href="{% static 'hqwebapp/js/lib/select2/select2.css' %}" rel="stylesheet"/>
+{% endblock %}
 {% block js %}{{ block.super }}
     <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
     <script src="{% static 'hqwebapp/js/lib/underscore-1.4.4.js' %}"></script>
     <script src="{% static 'app_manager/js/report-module.js' %}"></script>
+    <script src="{% static 'hqwebapp/js/lib/select2/select2.js' %}"></script>
 {% endblock %}
 {% block js-inline %}{{ block.super }}
     <script>
-        $(function () {
-            var saveURL = "{% url "corehq.apps.app_manager.views.edit_report_module" domain app.id module.id %}";
-            ko.applyBindings(new ReportModule({
-                moduleName: {{ module.name|JSON }},
-                availableReports: {{ all_reports|JSON }},
-                currentReports: {{ current_reports|JSON }},
-                saveURL: saveURL,
-                lang: {{ lang|JSON }}
-            }), $("#report-module-edit")[0]);
+        var saveURL = "{% url "corehq.apps.app_manager.views.edit_report_module" domain app.id module.id %}";
+        var reportModule = new ReportModule({
+            moduleName: {{ module.name|JSON }},
+            availableReports: {{ all_reports|JSON }}, // structure for all reports
+            currentReports: {{ current_reports|JSON }}, // config data for app reports
+            saveURL: saveURL,
+            lang: {{ lang|JSON }}
         });
+        ko.applyBindings(reportModule, $("#report-module-edit")[0]);
+
+        var select2s = $('.choice_filter');
+        for(var i = 0; i < select2s.length; i++) {
+            var element = select2s.eq(i);
```

Comment:
my syntax was bad - element isn't the jQuery object, so you'd also need `var $element = $(element);`

(not expecting you to change anything...it was just bothering me to not know why that didn't work)